tmt/steps/provision/__init__.py

```diff
@@ -170,7 +171,7 @@ def requires(self):
         return Guest.requires()
 
 
-class Guest(tmt.utils.Common):
```

Comment:
I will remove this, becasue of something my ide was blind to Common methods, in the origin state